arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/ModuleOperationStackedSet.java

```diff
@@ -15,8 +15,8 @@
 
 package net.consensys.linea.zktracer.container.stacked;
 
+import java.util.ArrayList;
```

Comment:
I have no idea if ArrayList is the best, open to suggestions !

Reply:
Yes, `ArrayList` is the best smart array you can have. Just watch out for resizes because it resizes automatically so picking a good starting size is always good.

Reply:
Do agree, for this we need real life data (see my answer to your comment about the prefered constructor)

Reply:
cf https://github.com/Consensys/linea-tracer/issues/1394